### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by tokatoka**
just build it beforehand if it can't find the .so

### CODE REVIEW

**Comment by tokatoka** [comment]
@julihoh 
Please have a look at this when you are back 😄

**Comment by domenukk** [comment]
I _think_ this was fixed in #950 but let's see if it's still shaky there

File: .github/workflows/build_and_test.yml [hunk 1]
```diff
@@ -180,6 +180,8 @@ jobs:
       run: rustup toolchain install nightly --component rustfmt --component clippy --allow-downgrade
     - name: Add no_std toolchain
       run: rustup toolchain install nightly-x86_64-unknown-linux-gnu ; rustup component add rust-src --toolchain nightly-x86_64-unknown-linux-gnu
+    - name: Use nigntly
+      run: rustup default nightly
```
Comment (domenukk) [inline .github/workflows/build_and_test.yml:184]: Let's not default to nightly, well not find a range of bugs

Comment (tokatoka) [reply .github/workflows/build_and_test.yml:184]: ok

**Comment by julihoh** [comment]
Hey @tokatoka @domenukk . Thanks for the fix!

I'm confused as to why it would be required though.. can someone point me to the CI log that initially failed?

**Comment by tokatoka** [comment]
https://github.com/AFLplusplus/LibAFL/actions/runs/3717688162/jobs/6305294036
This one

**Pull request merged**